src/modules/tvshowAPI.js

```diff
@@ -0,0 +1,9 @@
+const showsAPI = 'https://api.tvmaze.com/shows';
```

Comment:
- The web app uses an API that bases its development around it ✔
- The API should allow for getting a list of items with a unique item id and detailed information about the items. ✔
 
Well done 🙌🔥